ERCS/erc-7280.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: 7280
+title: NFT Metadata Extension like JSON-LD
+description: Let NFT metadata have a feature equivalent to JSON-LD to be semantic.
+author: Yohei Nishikubo (@yoheinishikubo)
+discussions-to: https://ethereum-magicians.org/t/eip-7280-nft-metadata-extension-like-json-ld/14935
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-04
+requires: 721, 1155, 3525
+---
+
+## Abstract
+
+This proposal expands the metadata format for Non-Fungible Tokens ([ERC-721](./eip-721.md), [ERC-1155](./eip-1155.md), [ERC-3525](./eip-3525.md), and others), adding support for linked data like JSON-LD format. The additional data is stored under the linked_data key in the metadata JSON.
+
+## Motivation
+
+The existing metadata format for Non-Fungible Tokens is limited and doesn't support the inclusion of structured and semantically meaningful data. By integrating JSON-LD (Linked Data), we can enhance the richness and interoperability of the metadata associated with NFTs.
+
+This allows for complex metadata structures that can link to external schemas and data, improving the contextual relevance and usability of NFTs across various applications.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+The JSON-LD based metadata is stored under a new `linked_data` key in the metadata JSON. The `linked_data` key is an array of objects, where each object contains two keys: `schema` and `data`.
+
+| name   | compliance level | type   | description                    |
+| ------ | ---------------- | ------ | ------------------------------ |
+| schema | MUST             | object | The schema of the linked data. |
+| data   | MUST             | object | The data of the linked data.   |
+
+### Schema
+
+| name        | compliance level | type   | description                    |
+| ----------- | ---------------- | ------ | ------------------------------ |
+| uri         | MUST             | string | The URI of the schema.         |
+| name        | MUST             | string | The name of the schema.        |
+| description | OPTIONAL         | string | The description of the schema. |
```

Comment:
It might be nice to use something like JSON Schema to specify these requirements. That's up to you though.